packages/core/package.json

```diff
@@ -120,6 +120,7 @@
   "devDependencies": {
     "@scion-scxml/test-framework": "^2.0.15",
     "@xstate-repo/jest-utils": "*",
+    "ajv": "^8.12.0",
```

Comment:
We were implicitly relying on `ajv@6.x`, we never specified what version of `ajv` we want to actually use.